.drone.yml

```diff
@@ -100,6 +100,7 @@ pipeline:
       - TEST_SERVER_URL=http://server/
       - TEST_SERVER_FED_URL=http://federated/
       - OC_TEST_ON_OBJECTSTORE=1
+      - BEHAT_FILTER_TAGS=~@app-required&&~@skip&&~@masterkey_encryption
```

Comment:
This works, but there is a bit of knowledge needed in the app ``.drone.yml`` to know what tags NOT to run.
Perhaps it would be easier/better in the long-term if, in ``core``, we positively tag all test scenarios/features that form the "defined base set of good core scenarios" - those that do not need some other app, are not skipped (because of some bug), do not mess with encryption... Then other apps can explicitly set the tag to ``@base``